tcms/testruns/static/testruns/js/get.js

```diff
@@ -46,13 +46,25 @@ $(document).ready(() => {
 
     $('.change-assignee-bulk').click(changeAssigneeBulk)
 
+    $('.bulk-change-status').click(function () {
+        // `this` is the clicked link
+        const statusId = $(this).data('status-id')
+        changeStatusBulk(statusId)
+
+        // close dropdown
+        $('.statuses-dropdown').removeClass('open')
+
+        return false
```

Comment:
You may want to add a comment why we need `return false` b/c it wasn't immediately obvious to me

Reply:
https://github.com/kiwitcms/Kiwi/pull/1967